packages/salesforcedx-vscode-core/test/commands/forceOrgCreate.test.ts

```diff
@@ -0,0 +1,77 @@
+import { expect } from 'chai';
+import * as path from 'path';
+import * as sinon from 'sinon';
+import * as vscode from 'vscode';
+import {
+  AliasGatherer,
+  DEFAULT_ALIAS,
+  forceOrgCreate,
+  ForceOrgCreateExecutor
+} from '../../src/commands/forceOrgCreate';
+import { nls } from '../../src/messages';
+
+// tslint:disable:no-unused-expression
+describe('Force Org Create', () => {
+  describe('Alias Gatherer', () => {
+    const TEST_ALIAS = 'testAlias';
+    let inputBoxSpy: sinon.SinonStub;
+
+    before(() => {
+      inputBoxSpy = sinon.stub(vscode.window, 'showInputBox');
```

Comment:
Create a stub for InputBox. I want to simulate the 3 possible responses of a user hitting Escape (returns undefined), a user hitting Enter, and a user typing in an alias